CHANGELOG.md

```diff
@@ -405,30 +420,31 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - An initial version.
 
-[0.18.0]: https://github.com/aicers/review-web/compare/0.17.0...0.18.0
-[0.17.0]: https://github.com/aicers/review-web/compare/0.16.0...0.17.0
-[0.16.0]: https://github.com/aicers/review-web/compare/0.15.0...0.16.0
-[0.15.0]: https://github.com/aicers/review-web/compare/0.14.5...0.15.0
-[0.14.5]: https://github.com/aicers/review-web/compare/0.14.4...0.14.5
-[0.14.4]: https://github.com/aicers/review-web/compare/0.14.3...0.14.4
-[0.14.3]: https://github.com/aicers/review-web/compare/0.14.2...0.14.3
-[0.14.2]: https://github.com/aicers/review-web/compare/0.14.1...0.14.2
-[0.14.1]: https://github.com/aicers/review-web/compare/0.14.0...0.14.1
-[0.14.0]: https://github.com/aicers/review-web/compare/0.13.1...0.14.0
-[0.13.1]: https://github.com/aicers/review-web/compare/0.12.0...0.13.1
-[0.13.0]: https://github.com/aicers/review-web/compare/0.12.0...0.13.0
-[0.12.0]: https://github.com/aicers/review-web/compare/0.11.0...0.12.0
-[0.11.0]: https://github.com/aicers/review-web/compare/0.10.0...0.11.0
-[0.10.0]: https://github.com/aicers/review-web/compare/0.9.1...0.10.0
-[0.9.1]: https://github.com/aicers/review-web/compare/0.9.0...0.9.1
-[0.9.0]: https://github.com/aicers/review-web/compare/0.8.1...0.9.0
-[0.8.1]: https://github.com/aicers/review-web/compare/0.8.0...0.8.1
-[0.8.0]: https://github.com/aicers/review-web/compare/0.7.0...0.8.0
-[0.7.0]: https://github.com/aicers/review-web/compare/0.6.0...0.7.0
-[0.6.0]: https://github.com/aicers/review-web/compare/0.5.0...0.6.0
-[0.5.0]: https://github.com/aicers/review-web/compare/0.4.1...0.5.0
-[0.4.1]: https://github.com/aicers/review-web/compare/0.4.0...0.4.1
-[0.4.0]: https://github.com/aicers/review-web/compare/0.3.0...0.4.0
-[0.3.0]: https://github.com/aicers/review-web/compare/0.2.0...0.3.0
-[0.2.0]: https://github.com/aicers/review-web/compare/0.1.0...0.2.0
-[0.1.0]: https://github.com/aicers/review-web/tree/0.1.0
+[Unreleased]: https://github.com/aicers/review-web/compare/0.18.0...main
+[0.18.0]: <https://github.com/aicers/review-web/compare/0.17.0...0.18.0>
```

Comment:
I was wondering if the first `<` and last `>` are required. I remember they were sometimes put in our past code or sometimes  not.

Reply:
@minshao, these changes are from commit a40e23ac. Please create a PR to fix them. Once that has been merged to this branch, I'll rebase the branch and squash the fix with a40e23ac.

Reply:
@sehkone, thanks for pointing this out. Fixed the commit.